query-engine/connectors/sql-query-connector/src/ordering.rs

```diff
@@ -3,84 +3,124 @@ use connector_interface::QueryArguments;
 use prisma_models::*;
 use quaint::ast::*;
 
+static ORDER_JOIN_PREFIX: &str = "orderby_";
+
+#[derive(Debug, Clone)]
+pub struct AliasedJoin {
+    // Actual join data to be passed to quaint
+    pub(crate) data: JoinData<'static>,
+    // Alias used for the join. eg: LEFT JOIN ... AS <alias>
+    pub(crate) alias: String,
+}
+#[derive(Debug, Clone)]
+pub struct OrderingJoins {
+    // Joins necessary to perform the order by
+    pub(crate) joins: Vec<AliasedJoin>,
+    // Final column identifier to be used for the scalar field to order by
+    pub(crate) order_column: Column<'static>,
+}
```

Comment:
The `ordering::build` method returns the following data. I added the `order_column` in there so that the `cursor_condition` doesn't have to recompute it

Reply:
Great, thanks!